source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -0,0 +1,334 @@
+bindings. Other OVN-controllers notice the binding change, and update their
+local flows.
+
+## Packaging and installation
+
+### OVN Central Server
+The OVN central server must be intalled manually. It must be accessible via network to
+the OVN provider and to all the hosts which use OVN.
+
+### OVN Controller
+The OVN Controller must be installed on each host using OVN.
+After installation the OVN controller must be configured with the value of the OVN central
+which it will be using and the local IP used for tunneling.
+This could be automated using an otopi plugin, but this would require a one-time configuration
+of the plugin. A way to establish the tunneling IP on the host should be provided, as well as
+the IP of the OVN central.
+ A look at the plugin directory might be helpful to understand the process:
+/usr/share/ovirt-host-deploy/plugins/ovirt-host-deploy
+
+Another possibility would be to to use vdsm-tool configure.
+
+### Provider packaging and installation
+The provider will be delivered as an RPM. After installation the only configurable
+property will be the IP of the OVN north DB. If not specified, the provider will assume
+it is running on the same host as the provider is installed on.
+
+### Driver packaging
+The VIF driver will probably be delivered as an rpm. As such, it will be installed manually
+on the hosts. The VIF driver connects to the local OVS instance and OVN north DB instance used
+by the local OVS. It needs no further configuration.
+
+The VIF driver RPM could be made available in the VDSM repo, or a local repo available to the hosts. 
+It could then be installed by being added to:
+/usr/share/ovirt-host-deploy/plugins/ovirt-host-deploy/vdsmhooks/packages.d\
+All RPM files specified there will be installed on the host during the host deploy operation.
+There should be a separate file for each linux distribution. The file extension
+must be the name of the linux distribution on which the rpm's are to be installed
+(fedora, redhat, ...).
+
+## Further considerations
+
+### IPAM
+The IP assigned to OVN managed NICs should be assigned from a subnet (pool of IPs) definded within OVN.
+As of now, this is still not implemented in OVN.
+A request for this functionality has been created in the OVS bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1368043
+
+### Migration
+We must ensure a minimal NIC downtime during the live migration process. The switch over from the ports on the source and
+descination host must be as quick as possible and also synchronized with the switch over of the VM itself.
+The following BZ has been created to describe and track this issue:
+https://bugzilla.redhat.com/show_bug.cgi?id=1369362
+
+### High availability
+The current plan for high availability is to run northd on the Engine host, which can be highly-available via hosted engine. 
+We hope that OVN gives us high availability so that northd can be run on any chassis.
+
+## Testing
+
```

Comment:
also need to be tested:
* installation
* security: only Engine should be able to control our provider
* security: chassis connect to northd using a secure connection.